Automattic/wp-calypso
# Reverse error introduced in #75777

Related to https://github.com/Automattic/wp-calypso/pull/75777

## Proposed Changes

* After rebasing and merging https://github.com/Automattic/wp-calypso/pull/75777, I introduced an error which led to the PR being reverted.
* This PR aims to redeploy https://github.com/Automattic/wp-calypso/pull/75777 without said error. 
* The error has been fixed in https://github.com/Automattic/wp-calypso/commit/2b3190a895e92488a6906d8137afc9ed345019c3 

## Testing Instructions



* The test plan of https://github.com/Automattic/wp-calypso/pull/75777 can be followed
* Just note there have been some changes on trunk since #75777 was tested. For example, the screenshots in this [comment](https://github.com/Automattic/wp-calypso/pull/75777#issuecomment-1527863202) are no longer valid due to https://github.com/Automattic/wp-calypso/commit/e7903abc2f7c3a2dd3102885df6602b49bd7db36.

## Pre-merge Checklist
